tests/unit/modules/zypper_test.py

```diff
@@ -59,6 +60,28 @@ class ZypperTestCase(TestCase):
     Test cases for salt.modules.zypper
     '''
 
+    def setUp(self):
+        self.new_repo_config = dict(
+            name='mock-repo-name',
+            url='http://repo.url/some/path'
+        )
+        self.zypper_patcher_config = {
+            '_get_configured_repos': Mock(
+                side_effect=[
+                    Mock(**{'sections.return_value': []}),
+                    Mock(
+                        **{
+                            'sections.return_value': [
+                                self.new_repo_config['name']
+                            ]
+                        }
+                    ),
+                ],
+            ),
```

Comment:
@isbm your comment is obviously an exaggerations and I don't consider exaggerations as valid arguments.
I didn't find a better way to do this without breaking PEP8 (it is complaining about closing brackets not matching the visual indentation)
I also looked at this https://docs.saltstack.com/en/latest/topics/development/conventions/style.html#vertical-is-better maybe you would like to take a look as well
Expecially here: 

> For function definitions and function calls, Salt adheres to the PEP-8 specification of at most 80 characters per line.

I see calling `Mock()` as a 'function call' in this case